app.R

```diff
@@ -65,6 +68,54 @@ xlabel_view <- c(rep(c("black", "transparent", "transparent", "transparent"), 41
 #cc <- rev(c("#fde725", "#addc30", "#5ec962", "#28ae80", "#21918c", "#2c728e", "#3b528b", "#472d7b", "#440154"))
 viridis_cc <- c("#440154", "#2c728e", "#28ae80", "#addc30")
 
+# ITCR Slido Data
+itcr_slido_data <- readRDS(file.path("data", "itcr_slido_data.RDS"))
+
+# Workshops Data
+poll_data <- itcr_slido_data$`Polls-per-user` %>%
+  janitor::clean_names()
+poll_data <- poll_data %>% 
+  mutate(merged_likely_rec = if_else(is.na(how_likely_would_you_be_to_recommend_this_workshop), how_likely_would_you_be_to_recommend_this_workshop_2, 
+                                     how_likely_would_you_be_to_recommend_this_workshop))
+
+poll_data_subset <- poll_data %>%
+  dplyr::filter(how_likely_are_you_to_use_what_you_learned_in_your_daily_work %in% c("Extremely likely", 
+                                                                                     "Likely",
+                                                                                     "Not very likely", 
+                                                                                     "Somewhat likely", 
+                                                                                     "Very likely"))
+
+poll_data_subset$how_likely_are_you_to_use_what_you_learned_in_your_daily_work <- factor(poll_data_subset$how_likely_are_you_to_use_what_you_learned_in_your_daily_work, 
+                                                                                         levels = c("Not very likely",  "Somewhat likely", "Likely", "Very likely", "Extremely likely"))
+
+# Wordcloud 
+ud_model <- udpipe::udpipe_load_model("wordcloud-model.udpipe")
+
+results <- udpipe::udpipe_annotate(ud_model, x = poll_data$what_did_you_like_most_about_the_workshop) %>%
+  as.data.frame() %>%
+  dplyr::filter(upos %in% c("NOUN", "ADJ", "ADV")) %>%
+  mutate(lemma= tolower(lemma)) %>%
+  count(lemma)
+
+rec_results <- udpipe::udpipe_annotate(ud_model, x = poll_data$please_share_any_recommendations_you_have_for_improvements) %>%
+  as.data.frame() %>%
+  filter(upos %in% c("NOUN", "ADJ", "ADV")) %>%
+  mutate(lemma= tolower(lemma)) %>%
+  count(lemma)
+
+# Collaborations
+collabs <- readr::read_tsv(file.path("data", "collabs.tsv")) %>% 
+  separate_rows("Category", sep = ", ", ) %>% 
+  mutate(Category = trimws(Category)) %>% 
+  filter(Category != "?")
+
+# Career Stage of Workshop Registrants
+career_stage_counts <- readr::read_tsv(file.path("data", "career_stage_counts.tsv"))
+career_stage_counts_subset <- career_stage_counts[c(1:5), c(1:11)]
```

Comment:
Can you give me a bit of insight on what you are trying to subset here? Can we do this by column names and filtering instead of by indexing? I'm worried if the data gets rearranged the indices may not be consistent.

Reply:
That was me, I was working from a [spreadsheet](https://docs.google.com/spreadsheets/d/1-8vox2LzkVKzhmSFXCWjwt3jFtK-wHibRAq2fqbxEyo/edit#gid=8290691) that has notes to the right and percents to the bottom of the counts I subset. I just subset it to have the workshop name, counts for each career stage, and the total lines.

Reply:
I was thinking of this PR as a draft version of the dashboard - it would contain placeholder plots with static datasets, but we wouldn't be using these datasets in the real version since we'll be automating the data collection using metricminer and GHA.